cookbook-react/recipes/inputs/barcelona/index.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import cn from 'classnames';
+
+import styles from './styles.module.scss';
+import { ReactComponent as ValidIcon } from './assets/ic_valid.svg';
+import { ReactComponent as InvalidIcon } from './assets/ic_invalid.svg';
+
+interface Props {
+  label?: string;
```

Comment:
Shouldn't this be required?
If not I'd render the `<label />` if `label` is undefined.

Reply:
Yeah you're right, I should've conditionally rendered the label. Although now that I think of it this input always has label (at least that's how the design is) so there's no point in making it optional. I'll just make it required